ibu-imager/ops/ops.go

```diff
@@ -124,3 +106,88 @@ func (o *ops) GetImageFromPodDefinition(podFile, containerImage string) (string,
 
 	return "", fmt.Errorf("no 'etcd' container found or no image specified in YAML definition: %w", err)
 }
+
+func (o *ops) RunUnauthenticatedEtcdServer(etcdImage, authFile string) error {
+	o.log.Info("Run unauthenticated etcd server for recert tool")
+
+	command := "podman"
+	args := []string{
+		"run", "--name", "recert_etcd",
+		"--detach", "--rm", "--network=host", "--privileged", "--replace",
```

Comment:
I think there are pretty much same params here and in recert, lets make some list for common?